libjcat/jcat-context.c

```diff
@@ -50,7 +50,7 @@ static void
 jcat_context_init (JcatContext *self)
 {
 	JcatContextPrivate *priv = GET_PRIVATE (self);
-	priv->localstatedir = g_strdup (JCAT_LOCALSTATEDIR);
+	priv->keyring_path = g_build_filename (JCAT_LOCALSTATEDIR, "lib", PACKAGE_NAME, NULL);
```

Comment:
Hmm but `PACKAGE_NAME` is the library name?  So potentially if something fails to set the new keyring path we end up using `/var/lib/libjcat`?  Maybe it's better to require that to be set in the API.

Reply:
It's just a sane default. In jcat-tool I actually used the user datadir as the sane default, maybe that makes sense to use here as well?

Reply:
e.g.

    -       priv->keyring_path = g_build_filename (JCAT_LOCALSTATEDIR, "lib", PACKAGE_NAME, NULL);
    +       priv->keyring_path = g_build_filename (g_get_user_data_dir (), PACKAGE_NAME, NULL);

Reply:
Yeah that's a better sane default I think.

Reply:
Okay, fixed.